Library/Formula/bazaar.rb

```diff
@@ -27,5 +28,23 @@ def install
     libexec.install 'bzr', 'bzrlib'
 
     bin.install_symlink libexec+'bzr'
+
+    # install the plugins under /var/bazaar/plugins/
+    plugins_orig = libexec/"bzrlib/plugins"
+    plugins_new = var/"bazaar/plugins"
+
+    Dir[plugins_orig/"*"].each do |plugin|
+      path = Pathname.new plugin
+      plugins_new.install plugin unless File.exist? (plugins_new/path.basename)
+    end
+
+    rm_rf plugins_orig
+    ln_s plugins_new, plugins_orig
+  end
```

Comment:
Try and `brew install ./bazaar-2.6.0_1.yosemite.bottle.tar.gz` and see if it puts things where you'd expect (e.g. that the contents of `var` aren't missing)